src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -199,6 +207,13 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
         /// <param name="keyData">The data view containing the terms. If specified, this should be a single column data
         /// view, and the key-values will be taken from that column. If unspecified, the key-values will be determined
         /// from the input data upon fitting.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[MapValueToKey](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversions/MapKeyToValueMultiColumn.cs)]
```

Comment:
>KeyToValue [](start = 127, length = 10)

ValueToKey

Reply:
i had them on the same file, at some point :)

---
In reply to: [271927396](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271927396) [](ancestors = 271927396)

Reply:
i do not see the change ..  i.e use  of MapValueToKeyMultiColumn.cs  for this API instead of  MapKeyToValueMultiColumn.cs

---
In reply to: [271930498](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271930498) [](ancestors = 271930498,271927396)